gse-app/src/main/java/com/powsybl/gse/app/ProjectPane.java

```diff
@@ -56,6 +56,8 @@
 
     private static final ServiceLoaderCache<ProjectFileExecutionTaskExtension> EXECUTION_TASK_EXTENSION_LOADER = new ServiceLoaderCache<>(ProjectFileExecutionTaskExtension.class);
 
+    private int tabsOpen = 0;
```

Comment:
I'm not sure this counter is needed: the number of tabs can probably be retrieved elsewhere